CONTRIBUTING_MODELS.md

```diff
@@ -0,0 +1,235 @@
+class, `DistilBertTokenizer`, which inherits from `keras_nlp.tokenizers.WordPieceTokenizer`.
+All the underlying actual tokenization will be taken care of by the superclass.
+
+The important thing here is adding "special tokens". Most models have
+special tokens such as beginning-of-sequence token, end-of-sequence token,
+mask token, pad token, etc. These have to be
+[added as member attributes](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/distil_bert/distil_bert_tokenizer.py#L91-L105)
+to the tokenizer class. These member attributes are then accessed by the
+preprocessor class.
+
+For a full list of the tokenizers KerasNLP offers, please visit [this link](https://keras.io/api/keras_nlp/tokenizers/) and make use of the tokenizer your model uses!
+
+#### Preprocessor
+
+The preprocessor class is responsible for making the inputs suitable for consumption
+by the model - it packs multiple inputs together, i.e., given multiple input texts,
+it will add appropriate special tokens, pad the inputs and return the dictionary
+in the form expected by the model.
+
+The preprocessor class might have a few intricacies depending on the model. For example,
+the DeBERTaV3 tokenizer does not have the `[MASK]` in the provided sentencepiece
+proto file, and we had to make some modifications [here](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/deberta_v3/deberta_v3_preprocessor.py). Secondly, we have
+a separate preprocessor class for every task. This is because different tasks
+might require different input formats. For instance, we have a [separate preprocessor](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/distil_bert/distil_bert_masked_lm_preprocessor.py)
+for masked language modeling (MLM) for DistilBERT. So, we don't always recommend
+adding the preprocessor right away; this can optionally be done in step 5.
+
+#### Unit Tests
+
+The last step here is to add unit tests for both the tokenizer and the preprocessor.
+A dummy vocabulary is created, and the output of both these layers is verified
+including tokenization, detokenization, etc.
+
+### Step 4: PR #3 - Add XX Presets
+
+Once the two PR have been merged, you can open a PR for adding presets. For every
+model, we have a separate file where we mention our preset configurations.
+This preset configuration has model-specific arguments such as number of layers,
+number of attention heads; preprocessor-specific arguments such as whether we want to
+lowercase the input text; checkpoint and vocabulary file URLs, etc.
+In the PR description, you can add Google Drive/personal GCP bucket links to the
+checkpoint and the vocabulary files. These files will then be uploaded to GCP by us!
+
+After wrapping up the preset configuration file, you need to
+add the `from_preset` function to all three classes, i.e., `DistilBertBackbone`,
+`DistilBertTokenizer` and `DistilBertPreprocessor`,. Here is an
```

Comment:
remove trailing comma, and potentially `DistilBertPreprocessor` as mentioned above.